tool/tsh/common/tsh.go

```diff
@@ -2086,14 +2086,36 @@ func onLogout(cf *CLIConf) error {
 		proxyHost = cf.Proxy
 	}
 
-	switch {
-	// Proxy and username for key to remove.
-	case proxyHost != "" && cf.Username != "":
-		tc, err := makeClient(cf)
+	tc, err := makeClient(cf)
```

Comment:
Will this work if I specify exactly one of `proxyHost` and `cf.Username`? We had a deliberate user friendly message in that case before.